README.md

```diff
@@ -25,6 +25,38 @@ Once it is available in your Cloud Foundry environment, configure your applicati
 
 ### Configuration
 
+#### Unified Service Tagging
+
+> This feature requires the Datadog Cluster Agent to be installed. 
+See [Datadog Cluster Agent BOSH Release](https://github.com/DataDog/datadog-cluster-agent-boshrelease).
+
+Unified service tagging ties Datadog telemetry together through using three reserved tags: `env`, `service`, and `version`. In Cloud Foundry, they are set through the application labels/annotations.
+
+```yaml
+ env:
+    DD_ENV: <ENV_NAME>
+    DD_SERVICE: <SERVICE_NAME>
+    DD_VERSION: <VERSION>
+  metadata:
+    labels:
+      tags.datadoghq.com/env: <ENV_NAME>
+      tags.datadoghq.com/service: <SERVICE_NAME>
+      tags.datadoghq.com/version: <VERSION>
+```
+
+The `tags.datadoghq.com` prefix is part of the Agent Autodiscovery notation as described in [Basic Agent Autodiscovery Documentation](https://docs.datadoghq.com/getting_started/containers/autodiscovery).
```

Comment:
Can we also mention how this autodiscovery prefix is a way to set "normal" tags as well?